doc/examples/moveit_cpp/moveitcpp_tutorial.rst

```diff
@@ -29,4 +29,50 @@ The entire code can be seen :codedir:`here in the MoveIt GitHub project<examples
 
 The Launch File
 ---------------
-The entire launch file is :codedir:`here<examples/moveit_cpp/launch/moveit_cpp_tutorial.launch.py>` on GitHub. All the code in this tutorial can be run from the **moveit2_tutorials** package that you have as part of your MoveIt setup.
+The entire launch file is :codedir:`here<examples/moveit_cpp/launch/moveit_cpp_tutorial.launch.py>` on GitHub.
+Notably, the launch file contains the following node
+
+.. code-block:: python
+
+    # MoveItCpp demo executable
+        moveit_cpp_node = Node(
+            name="moveit_cpp_tutorial",
+            package="moveit2_tutorials",
+            executable="moveit_cpp_tutorial",
+            output="screen",
+            parameters=[moveit_config.to_dict()],
+        )
+This node contains parameters which are passed to the executable associated with moveit_cpp_tutorial.
```

Comment:
nit: Add a blank line in between these 2 -- although it seems to render fine as is